Dockerfile

```diff
@@ -20,12 +19,14 @@ RUN rm -rf ${ANDROID_DIR}/tools \
 # Required to accept licenses:
 # https://stackoverflow.com/questions/38096225/automatically-accept-all-sdk-licences/42125740#42125740
 # There will be a bit of delay (approx 30 seconds)
```

Comment:
Can you change this to mention that the download takes place during this step and may take up to a couple of minutes?

Reply:
I'm going to add that to the docs - there's no way to get this actually displayed that won't be hidden by the very same screen.